sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-processing-steps/validate-disbursement-base.ts

```diff
@@ -36,6 +36,18 @@ export abstract class ValidateDisbursementBase {
       log.info(`Student MSFAA associated with the disbursement is not signed.`);
       shouldContinue = false;
     }
+    // Disability Status PD/PPD Verified.
+    if (
```

Comment:
I would recommend taking the condition out and setting it to boolean variable. And also using true instead of null.

![image](https://github.com/bcgov/SIMS/assets/54600590/0701a804-4b7f-49ee-80b2-34a8a9c6332a)